src/cps/loader.cpp

```diff
@@ -184,20 +190,15 @@ namespace cps::loader {
         get_components(const Json::Value & parent, std::string_view parent_name, const std::string & name) {
             Json::Value compmap;
             if (!parent.isMember(name)) {
-                return tl::unexpected(fmt::format("Required field Components of {} is missing!", parent_name));
+                return tl::unexpected(fmt::format("Required field `components` of `{}` is missing!", parent_name));
             }
 
             std::unordered_map<std::string, Component> components{};
 
             // TODO: error handling for not an object
```

Comment:
@dcbaker What error handling did you expect? Something different than constructing an unexpected object?